tools/eslint-plugin/dist/configs/itwinjs-recommended.js

```diff
@@ -50,7 +50,14 @@ module.exports = {
     "@typescript-eslint/explicit-module-boundary-types": "off",
     "@typescript-eslint/indent": [
       "error",
-      2
+      2,
+      // SEE: https://eslint.org/docs/latest/rules/indent#options
+      {
+        // not the above documented default (0) but seems to be the default in our version of @typescript-eslint/indent
+        SwitchCase: 1,
```

Comment:
Why did you need to change this?

Reply:
specifically chained ternary expressions would keep indenting, going off memory something like this:
```ts
const w = a ? x
                   : b ? y
                        : z;
```

Reply:
spoke to Caleb, removing this change atm

Reply:
going to do a stricter check, but `rush lint` was stalling for me (last time that happened due to having a 10GB file in my git index, which I don't this time), so I only linted common, transformer, backend, and bentley. Going to double check all affected packages.